tibiawikisql/utils/charms.json

```diff
@@ -0,0 +1,98 @@
+[
+  {
+    "name": "Wound",
+    "type": "Offensive",
+    "description": "Wounds the creature and deals 5% of its initial hit points as Physical Damage.",
+    "points": "600 points"
```

Comment:
If the field is called "points", why not remove "points" from the content, so the field can be an integer and not a string?